src/style.css

```diff
@@ -1,3 +1,55 @@
 body {
+  font-family: Roboto, sans-serif;
   margin: 0;
 }
+
+header {
+  display: flex;
+  padding: 0 50px;
+  align-items: center;
+  justify-content: space-between;
+}
+
+.logo__img {
+  display: block;
+  width: 40px;
+  height: 40px;
+}
+
+.nav__list {
+  display: flex;
+  margin: 0;
+  padding: 0;
+  list-style: none;
+}
+
+.nav__link {
+  display: block;
+  position: relative;
+  height: 60px;
+  line-height: 60px;
+  font-size: 12px;
+  margin-left: 20px;
+  color: #000;
+  text-decoration: none;
+  text-transform: uppercase;
+}
+
+.is-active {
+  color: #00acdc;
+  margin-left: 0;
```

Comment:
redundant 
```suggestion
```

Reply:
https://github.com/mate-academy/layout_moyo-header/pull/3941/files#r1352628998
delete this please